### DISCUSSION TITLE
outline panel availability?

### DISCUSSION THREAD

**Discussion by zachvalenta** [zed-industries/zed#13421] [Feature Requests]
seems like it's in progress:

![Screenshot 2024-06-23 at 11 39 03 AM](https://github.com/zed-industries/zed/assets/11388735/3048c72e-cf90-44b1-9a85-c217d1f22cf3)

but not currently available (on the Apple Silicon version, at least):

![Screenshot 2024-06-23 at 11 37 56 AM](https://github.com/zed-industries/zed/assets/11388735/c4c94a33-9ba4-45ab-a89e-f68da55c1d25)

**Comment by SomeoneToIgnore** [comment]
Good point, it's not in the system menu — welcome to create a PR to add it there.
So far though, it's not really finished, but still available in Zed Preview (0.141) and quite easy to find near the project panel button, in the bottom left corner:

![image](https://github.com/zed-industries/zed/assets/2690773/716f64ce-2a5a-4caf-b9d2-b463bf4cf81b)

**Reply by zachvalenta** [reply]
thanks for info, appreciate it!
